resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -27,6 +27,14 @@ set +e
 uidentry=$(getent passwd $myuid)
 set -e
 
+if [ -n "$HADOOP_CONFIG_URL" ]; then
```

Comment:
Would it work to have a conditional wrapping this one to see if the `$HADOOP_HOME/conf` exists and contains the config files, then if not, check the URL? Or, are you forcing an update to the correct versions of the files whether they exist or not?

Reply:
Yes I just update them if the url exists for now. I am also waiting for work that @ifilonenko is doing. This is temporary anyway.